public/app/pages/routes.ts

```diff
@@ -1,5 +1,5 @@
 export enum ROUTES {
-  CONTINOUS_SINGLE_VIEW = '/',
+  CONTINUOUS_SINGLE_VIEW = '/',
```

Comment:
Actually since there's no "adhoc" pages, the `CONTINUOUS` prefix is not needed, so maybe we should just call it `SINGLE_VIEW`?

Reply:
Sounds good. Updated.